app/players/elves/druid.py

```diff
@@ -0,0 +1,23 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one,